concourse/tasks/run-db-migrations.yml

```diff
@@ -0,0 +1,25 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: govuk/ecs-cli
+    tag: latest
+    username: ((docker_hub_username))
+    password: ((docker_hub_authtoken))
+inputs:
+  - name: govuk-infrastructure
+    path: src
+  - name: app-terraform-outputs
+  - name: task-definition-arn
+  - name: run-task-command
+    optional: true
```

Comment:
These two lines are unnecessary since we only provide the CMD for the db migrations tasks through the param `COMMAND` below.

Reply:
Actually, I think you can delete this file, and add `DISABLE` to the params of the current `run-task.yml`  file.

Reply:
Im following you, yup thats a good idea, it makes it all generic rather than having an ackward almost duplication of run-task.yml file.